
# Get the information of the part held by the robot

---

bool success
string message

Part part
